packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.tsx

```diff
@@ -0,0 +1,237 @@
+import * as React from 'react';
+import * as stylesImport from './DetailsHeader.scss';
+const styles: any = stylesImport;
+import { Icon } from '../../Icon';
+import {
+  BaseComponent,
+  css,
+  IRenderFunction,
+  createRef,
+  IDisposable
+} from '../../Utilities';
+import { IColumn, ColumnActionsMode } from './DetailsList.types';
+
+import { ITooltipHostProps } from '../../Tooltip';
+import {
+  IDragDropHelper,
+  IDragDropOptions,
+} from './../../utilities/dragdrop/interfaces';
+
+const INNER_PADDING = 16; // Account for padding around the cell.
+const ISPADDED_WIDTH = 24;
+
+export interface IDetailsColumnProps extends React.Props<DetailsColumn> {
+  componentRef?: () => void;
+  column: IColumn;
+  columnIndex: number;
+  parentId?: string;
+  onRenderColumnHeaderTooltip?: IRenderFunction<ITooltipHostProps>;
+  onColumnClick?: (ev: React.MouseEvent<HTMLElement>, column: IColumn) => void;
+  onColumnContextMenu?: (column: IColumn, ev: React.MouseEvent<HTMLElement>) => void;
+  dragDropHelper?: IDragDropHelper | null;
+  isDraggable?: boolean;
+  setDraggedItemIndex?: (itemIndex: number) => void;
```

Comment:
>setDraggedItemIndex?: (itemIndex: number) => void; [](start = 2, length = 50)

Rather than this, the notification should be simple - like 'onColumnDragStart' with no params. The params (index) can be handled inside the DetailsHeader, where this callback is binded to a function.